src/helper.js

```diff
@@ -176,6 +177,14 @@ function passwordCompare(password, expected) {
 	return bcrypt.compare(password, expected);
 }
 
+function getDefaultNick() {
+	if (!this.config.defaults.nick) {
+		return "thelounge";
+	}
```

Comment:
Is this default not supposed to come from our own default copy of the config file?

Reply:
This is a fail-safe if user overrides the default with empty string.